permissions/UserDirs.permission

```diff
@@ -0,0 +1,24 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-userdirs
+# x-sailjail-description = User directories
+# x-sailjail-translation-key-long-description = permission-la-userdirs_description
+# x-sailjail-long-description = Access common user directories
+
+include /etc/sailjail/permissions/Documents.permission
+
+include /etc/sailjail/permissions/Downloads.permission
+
+include /etc/sailjail/permissions/Music.permission
+
+include /etc/sailjail/permissions/Pictures.permission
+
+mkdir     ${HOME}/Public
```

Comment:
While I'm not totally sure if we should have this directory as a permission, but perhaps, I think in any case it should be a separate permission that is then just included here. That way application can choose to have this permission without access to any of the other directories. The same applies to Templates directory.

Reply:
Like 3730ce50bde007216149e30de9010c5f6fc14a3a ?

Reply:
Yes

Reply:
Ok, I'll squash that then to the main commit and wait for final verdict.

Reply:
Done.

Reply:
Oh, sorry, I forgot about the latest change to this: You need to add also noblacklist rule for both directories and then blacklist it in Base.permission ([see the comment at the end](https://github.com/sailfishos/sailjail-permissions/commit/76e8aa5cfb1b279d9a53f1c13842ebc57de812f4#diff-cbb6d3ce8d0bd38db5cb3a5cf84c67aa0c8a7130efb833a164de2613ec01cabfR171)). That way application sees read-only version of the directory unless it has the needed permission.

Reply:
Added similar changes with 79cadf1 and updated `README.md` as well.